### PR TITLE
changes for switching to node6.9.0

### PR DESCRIPTION
**Pull request by madanagopaltcomcast**

### CODE REVIEW

File: examples/pxScene2d/src/rcvrcore/http_wrap.js [hunk 1]
```diff
@@ -18,470 +18,12 @@ limitations under the License.
 
 'use strict';
 
-var isV8=(typeof _isV8 !== "undefined");
-
+var isHttp2Present = false;
```
Comment (npoltorapavlo) [inline examples/pxScene2d/src/rcvrcore/http_wrap.js]: Please revert all changes to http_wrap (s). You are readding some old file which is out of date and doesn't have many other commits. http_wrap.js should work with node 6 and node 8, what is the problem you have with it?

File: examples/pxScene2d/src/CMakeLists.txt [hunk 2]
```diff
@@ -288,6 +288,24 @@ elseif (WIN32)
     set(PXCORE_LIB_LOCATION ${PXCOREDIR}/build/win/Release)
     set(PXSCENE_LINK_DIRECTORIES ${PXSCENE_LINK_DIRECTORIES} ${PXCORE_LIB_LOCATION})
     set(PLATFORM_SOURCES win/pxContextUtils.cpp)
+	if ((SUPPORT_NODE) AND (NOT USE_NODE_8))
```
Comment (npoltorapavlo) [inline examples/pxScene2d/src/CMakeLists.txt]: These lines had been removed as redundant. Are they really needed for node 6?

Comment (madanagopaltcomcast) [reply examples/pxScene2d/src/CMakeLists.txt]: these files are not compiled and facing build errors for windows

File: examples/pxScene2d/src/rcvrcore/http_wrap.js [hunk 3]
```diff
@@ -52,7 +54,7 @@ function Request(moduleName, appSceneContext, options, callback) {
   }
 
   var self = this;
-  var is_v2 = !isV8 && moduleName === 'http2' && http2;
+  var is_v2 = isHttp2Supported && !isV8 && moduleName === 'http2' && http2;
```
Comment (npoltorapavlo) [inline examples/pxScene2d/src/rcvrcore/http_wrap.js]: Please remove isHttp2Supported. http2 is checked.

**Pull request merged**